pkg/resource/parameter_group/sdk.go

```diff
@@ -247,14 +264,25 @@ func (rm *resourceManager) sdkUpdate(
 ) (updated *resource, err error) {
 	rlog := ackrtlog.FromContext(ctx)
 	exit := rlog.Trace("rm.sdkUpdate")
-	defer exit(err)
+	defer func() {
+		exit(err)
+	}()
 	if delta.DifferentAt("Spec.ParameterNameValues") {
 		ko, err := rm.resetParameterGroup(ctx, desired, latest)
 
 		if ko != nil || err != nil {
 			return ko, err
 		}
 	}
+
+	if delta.DifferentAt("Spec.Tags") {
+		err = rm.updateTags(ctx, desired, latest)
+	}
+
+	if err != nil {
+		return nil, err
+	}
```

Comment:
The error check here should be right after L279. Otherwise, if there is no difference at `Spec.Tag` it's gonna evaluate the error declared in L264